jax/_src/api.py

```diff
@@ -1118,9 +1118,8 @@ def hessian(fun: Callable, argnums: Union[int, Sequence[int]] = 0,
 def _std_basis(pytree):
   leaves, _ = tree_flatten(pytree)
   ndim = sum(map(np.size, leaves))
-  # TODO(mattjj): use a symbolic identity matrix here
   dtype = dtypes.result_type(*leaves)
-  flat_basis = np.eye(ndim, dtype=dtype)
+  flat_basis = jax._src.lax.lax._eye(dtype, (ndim, ndim), 0)
```

Comment:
Would it make sense just to use `jax.numpy.eye` here?

Reply:
confirmed, importing `jax.numpy` at the top of `api.py` results in:
```
E   AttributeError: partially initialized module 'jax' has no attribute 'custom_jvp' (most likely due to a circular import)
```

Reply:
I changed it to use `jax.numpy.eye` without an explicit import.